listenbrainz_spark/postgres/release.py

```diff
@@ -0,0 +1,86 @@
+from listenbrainz_spark.path import RELEASE_METADATA_CACHE_DATAFRAME
+from listenbrainz_spark.postgres.utils import save_pg_table_to_hdfs
+
+
+def create_release_metadata_cache():
+    """ Import release metadata from postgres to HDFS for use in stats calculation. """
```

Comment:
It is rather kind of scary how much part of this query are just "ah, query bah for X. then meh for Y..." but this 80+ lime query is composed of things I've recently written.... heh.

Reply:
:)